contribs/gmf/src/directives/layertree.js

```diff
@@ -290,6 +290,7 @@ gmf.LayertreeController.prototype.updateLayerDimensions_ = function(layer, node)
       var value = this.dimensions[key];
       if (value !== undefined) {
         dimensions[key] = value;
+        node.dimensions[key] = value;
       }
```

Comment:
This if looks strange for me, for me it should be like this:
```
       var value = this.dimensions[key];
       dimensions[key] = value !== undefined ? value : node.dimensions[key];
```
@fredj what do you think?